global_helpers/panther_base_helpers.py

```diff
@@ -221,6 +221,15 @@ def crowdstrike_detection_alert_context(event: dict):
     }
 
 
+def slack_alert_context(event: dict):
+    return {
+        "actor_name": deep_get(event, "actor", "user", "name", default="<MISSING_NAME>"),
+        "actor_email": deep_get(event, "actor", "user", "email", default="<MISSING_EMAIL>"),
+        "actor_ip": deep_get(event, "context", "ip_address", default="<MISSING_IP>"),
+        "user-agent": deep_get(event, "context", "ua", default="<MISSING_UA>")
```

Comment:
nit: - vs _ ? not sure if that was intentional

Reply:
This was intentional -- I can change it to use `-` instead of `_`